frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.tsx

```diff
@@ -171,24 +188,52 @@ const DashboardApp = (props: DashboardAppProps) => {
     onTimeout,
   });
 
+  const { refreshDashboard } = useRefreshDashboard({
+    dashboardId: dashboardId,
+    parameterQueryParams,
+  });
+
   const {
-    documentTitle: _documentTitle,
-    pageFavicon: _pageFavicon,
-    isRunning: _isRunning,
-    isLoadingComplete: _isLoadingComplete,
-    children,
-    ...dashboardProps
-  } = props;
+    hasNightModeToggle,
+    hideDownloadButton,
+    hideParameters,
+    isFullscreen,
+    isNightMode,
+    onNightModeChange,
+    refreshPeriod,
+    onFullscreenChange,
+    setRefreshElapsedHook,
+    onRefreshPeriodChange,
+    theme,
+    titled,
+    font,
+  } = useDashboardUrlParams({ location, onRefresh: refreshDashboard });
+
+  useSyncURLSlug({ location });
```

Comment:
`useDashboardNav({ isFullscreen })` needs to be added here as well since it was a part of the `DashboardControl` HoC.

Reply:
Actually it seems like the bug Kelvin run into it's probably a race condition (that we repro-ed on master as weel).
It seems like the logic in `useDashboardNav` is not needed anymore as the navbar is not rendered when in fullscreen